antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/SqlBase.g4

```diff
@@ -106,7 +106,8 @@ statement
     | SELECT topClause? selectElements
     fromClause
     whereClause?
-    specialClause? #selectStatement
+    specialClause?
+    (WITHOUT NULL (ALL | ANY))? #selectStatement
     ;
```

Comment:
To keep the structure, rename this statement as `withoutNullClause` and put it into specialClause.